app/presenters/export/csv/local_authority_presenter_module.rb

```diff
@@ -11,6 +11,18 @@ def local_authority_name
     @project.local_authority.name
   end
 
+  def local_authority_contact_name
+    return if @project.contacts.where(category: "local_authority").empty?
+
+    @project.contacts.where(category: "local_authority").pluck(:name)
+  end
+
+  def local_authority_contact_email
+    return if @project.contacts.where(category: "local_authority").empty?
+
+    @project.contacts.where(category: "local_authority").pluck(:email)
```

Comment:
Can you add `.join(", ")` to the end of these to get rid of the array delimiters